danger-klaxit/lib/klaxit/config_sorter.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Klaxit
+  class ConfigSorter
+    attr_reader :file
+
+    # Regexp that detects a group, and returns its name as first match.
+    PARENT_RE = /\A(\w+):/.freeze
+    # Regexp that detects any child key in a group, and returns its name as
+    # first match.
+    CHILD_RE = /\A  (\w+):/.freeze
```

Comment:
I did first, but it was a bit harder to read, and mostly i asked myself: If we do so, shouldn't it be a user choice ? And since the klaxit choice is two tabs for now there's no need to go further IMHO.

Reply:
I agree. And this repository is a set of tools for our teams, so we don't need to be exhaustive here.